src/worker.cc

```diff
@@ -88,12 +88,12 @@ const std::unordered_map<unsigned, unsigned> ERROR_TO_STATUS{
 
     {120, 400}, {121, 400}, {122, 400}, {123, 400}, {124, 400}, {125, 400}, {126, 400},
 
-    {130, 400}, {131, 400}, {132, 400}, {133, 400},
+    {130, 400}, {131, 400}, {132, 400}, {133, 400}, {136, 400},
```

Comment:
are `134` and `135` reserved for something else?

Reply:
I added them. They were used in the Valhalla error list but were not in the OSRM list.